### ISSUE TITLE
Phase 1: Fix footer color contrast on doi.datacite.org, commons.datacite.org, schema.datacite.org

### ISSUE THREAD

**Issue by paulvierkant** [datacite/datacite#2458]
**Description**
Increase text/link/icon color contrast in all footer areas to meet WCAG 1.4.3/1.4.11.
This is a cross-site issue visible in every audit.

**Includes**
- doi.datacite.org:
	- [x] Low-contrast light-gray footer text on dark background
- commons.datacite.org
	- [x]  Low-contrast links
 	- [ ]  hover states
 - schema.datacite.org
 	- [x] Low-contrast links
 	- [ ] “pill” elements

**Acceptance Criteria**
- All footer text and links reach ≥ 4.5:1 contrast
- Hover/focus states maintain contrast
- No Lighthouse contrast errors on footers (all sites)

**Comment by jrhoads** [comment]
@paulvierkant For fabrica and commons, looking at the footer, is there a particular color that you would prefer for links?
Or are you ok with me making it more black since it is already a grey?

**Comment by paulvierkant** [comment]
Please use the color palette of the [design manual](https://datacite.org/wp-content/uploads/2024/06/DataCite_Design_manual-version_2_1.pdf) (p.28, 29). Most probably it would be our primary dark blue: #243B54 I guess.

**Comment by jrhoads** [comment]
> Please use the color palette of the [design manual](https://datacite.org/wp-content/uploads/2024/06/DataCite_Design_manual-version_2_1.pdf) (p.28, 29). Most probably it would be our primary dark blue: #243B54 I guess.

Will do.  I'll use #243B54

**Comment by jrhoads** [comment]
Footer has been updated in
- [doi.datacite.org](https://doi.datacite.org)
- [commons.datacite.org](https://commons.datacite.org)
